test/test_helper.exs

```diff
@@ -32,6 +32,30 @@ defmodule RpcEndpoint do
 end
 spawn(&RpcEndpoint.init/0)
 
+defmodule ExampleService do
+  use Gnat.Services.Server
+
+  def request(%{topic: "calc.add", body: body}, _endpoint, _group) when body == "foo" do
+    :timer.sleep(100)
```

Comment:
it looks like we are just using this delay to validate that we have a non-zero processing time. How about bumping it down to 10ms? Avoid slowing down tests, but still provide a non-zero latency?